lucene/benchmark/src/java/org/apache/lucene/benchmark/byTask/feeds/ReutersContentSource.java

```diff
@@ -146,4 +172,11 @@ public synchronized void resetInputs() throws IOException {
     nextFile = 0;
     iteration = 0;
   }
+
+  private synchronized void createThreadIndex() {
+    if (!threadIndexCreated) {
```

Comment:
`== false` instead?  Or maybe change to `assert threadIndexCreated == false` since you also check this up above with a real `if` already?

Reply:
Sure, will do the required changes.